src/library_webgl.js

```diff
@@ -2429,11 +2429,12 @@ var LibraryGL = {
       var view = GL.miniTempBufferFloatViews[4*count-1];
       // hoist the heap out of the loop for size and for pthreads+growth.
       var heap = HEAPF32;
-      for (var i = 0; i < 4*count; i += 4) {
-        view[i] = heap[value + 4*i >> 2];
-        view[i+1] = heap[value + 4*i+4 >> 2];
-        view[i+2] = heap[value + 4*i+8 >> 2];
-        view[i+3] = heap[value + 4*i+12 >> 2];
+      value >>= 2;
+      for (var i = 0; i < 4 * count; i += 4) {
+        view[i] = heap[value + i];
```

Comment:
Hmm, perhaps `var dst = value + i;`? (closure will not do common subexpression elimination)

Reply:
Sounds good, might also improve code clarity anyhow.